target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -143,6 +143,7 @@ ramips_setup_macs()
 		label_mac=$(mtd_get_mac_binary factory 0x4)
 		;;
 	linksys,e5600|\
+    linksys,ea6350-v4|\
```

Comment:
wrong indentation, also in the other base-files

Reply:
Updated.  Editor was using tab-to-spaces, which I didn't turn off.